Documentation/running.md

```diff
@@ -23,7 +23,7 @@ flanneld -subnet-file /vxlan.env -etcd-prefix=/vxlan/network
 
 1. Download a `flannel` binary.
 ```bash
-wget https://github.com/coreos/flannel/releases/download/v0.10.0/flanneld-amd64 && chmod +x flanneld-amd64
+wget https://github.com/coreos/flannel/releases/download/v0.12.0/flanneld-amd64 && chmod +x flanneld-amd64
```

Comment:
version v0.13.0 is already out. Could you please update with v0.13.0

Reply:
Fixed